### ISSUE TITLE
StepMania Complete CD-ROM questions

### ISSUE THREAD

**Issue by blueapple128** [stepmania/stepmania-site#68]
Not sure if this is the right place to ask this question, but the forum registration directed me here.

Is there any documentation on what the contents of the StepMania Complete CD-ROM were? ([archive link](http://web.archive.org/web/20130621072035/http://www.stepmania.com:80/wiki/StepMania_Complete_CD-ROM)) It hasn't been available for about five years now, but I'm trying to reconstruct it.

Unfortunately, it looks like any specific data on what the disc used to have is sparse. I'm hoping someone here knows more about it.

Looking at the archives of the page over the years, there seem to have been several versions of the disc with varying descriptions. Here's everything I've been able to find on my own:

**2006:**

Mentions one version with 39 songs. According to the FAQ this includes all songs listed on the "Download Songs" page. As of this year, that page has exactly 39 songs (23 from StepMix 1, and 16 not from a StepMix). So far so good.

**2007:**

Mentions three versions:

* StepMania Complete 1 with 39 songs
* StepMania Complete 2 with 128 songs
* StepMania Complete 1+2 with 164 songs

According to the FAQ the three versions are supposed to include

1\) the "Download Songs" page from last year
2\) the "Download StepMix Songs" page from this year
1+2) both pages together from this year.

Right away something's off, since the Download StepMix Songs page has only 97 songs. The Download Songs page (updated for this year) has 34 songs not from a StepMix. None of the 39 songs from last year have been removed, so at this time, the site has only ever hosted a total of 131 songs. What are the 33 mystery songs apparently added to SMC 1+2 to give it 164? What are the 31 songs apparently added to SMC 2 to give it 128? What's the difference between this 33 and this 31? I can find no information on any of these questions.

**2008-2009:**

Mentions three versions:

* StepMania Complete 1+2+3 with 213 songs
* StepMania Complete 3 with 94 songs
* StepMania Complete 2 with 98 songs

The FAQ says the three versions are supposed to include

2\) the "Download StepMix 1 Songs" page as of last year
3\) the "Download StepMix 2 Songs" page as of this year
1+2+3) the two StepMixes together plus "some of" the songs on the "Download Songs" page, as of this year.

This set is a mixed bag. Unlike last year, this time two of the descriptions actually look correct. The Download StepMix 1 and 2 pages include exactly 98 and 94 songs, so those versions are reconstructable. But there is no explanation for why SMC 2's description suddenly shrunk from 128 to 98 songs. Meanwhile, SMC 1+2+3's description implies that it includes the 192 StepMix songs, plus 21 songs not from a StepMix. But the "Download Songs" page lists 36 songs, with no explanation for which 21 are chosen.

**2010:**

Mentions two versions:

* StepMania Complete 2010 with "350+" songs
* StepMania Complete 1+2+3 with 200 songs

This is the weirdest one. SMC 1+2+3 has suddenly shrunk from 213 songs to 200 songs. Meanwhile a new version SMC 2010 has appeared out of nowhere. The FAQ is unchanged, so there is no explanation for SMC 1+2+3's change and no information on SMC 2010 at all, other than its song count. I cannot find a single place that even mentions what songs are on the disc. And what is the "+" supposed to mean? Either the disc has 350 songs or it doesn't; there shouldn't be any variation.

**2011:**

Mentions two versions:

* StepMania Complete 2010 with 350 songs
* StepMania Complete 1+2+3 with 200 songs

The + sign gets silently dropped from SMC 2010. There is still no explanation of what the 350 songs on the disc are.

**2012-2013:**

Mentions one version, StepMania Complete 2010 with 350 songs.

Looks like the site no longer sells the 1+2+3 version. There is still no explanation of what the 350 songs on the disc are.

**2014:**

The page to purchase the CD no longer exists on stepmania.com. At this point the contents of the disc seem to (hopefully not) be lost forever.

TL;DR there have been somewhere between six and nine versions of the CD over the years, and several of the descriptions contradict each other. Only the three smallest versions are possible to reconstruct, and the most complete ones don't have any description at all. Does anyone have any more information about the missing versions, or know someone else who would? Thanks in advance!

TL;DR 2: This is a breakdown of the known versions and missing information:

\- SMC 1: 39 songs. FAQ says "look at this page", which has exactly 39 songs.
**- SMC 2 ver. A: 128 songs. FAQ says "look at this page", but the page only has 97 songs.**
\- SMC 2 ver. B: 98 songs. FAQ says "look at this page", which has exactly 98 songs.
**- SMC 1+2: 164 songs. FAQ says "look at these pages", but the pages only have 131 songs.**
\- SMC 3: 94 songs. FAQ says "look at this page", which has exactly 94 songs.
**- SMC 1+2+3 ver. A: 213 songs. FAQ says "look at these pages", but the pages have 228 songs.**
**- SMC 1+2+3 ver. B: 200 songs. FAQ says "look at these pages", but the pages have 228 songs.**
**- SMC 2010 ver. A: 350+ songs. No description at all or what the + sign is supposed to mean.**
**- SMC 2010 ver. B: 350 songs. No description at all.**

There is also no information why some versions of the disc with the same name have changed their description over the years.

**Stuff I've tried already**:

- I found [this post on r/Stepmania](https://www.reddit.com/r/Stepmania/comments/71vzck/stepmania_complete_cd/) also asking about the disc, but the only comment is a known knockoff version that mentions 171 songs (no official version has this) as well as at least 3 official DDR songs (which of course the official version doesn't have).

- According to the current site FAQ and the archives of the old site, Chris Danford did 100% of the handling work for the disc. I've sent him a message on Twitter, but he doesn't seem to have been on since January.

- I've originally posted this question on [https://zenius-i-vanisher.com/v5.2/thread?threadid=10004](https://zenius-i-vanisher.com/v5.2/thread?threadid=10004), then cross-posted it to r/Stepmania ([post link](https://www.reddit.com/r/Stepmania/comments/94wkl7/stepmania_complete_cdrom_questions/)) and r/Etterna ([post link](https://www.reddit.com/r/Etterna/comments/97nod6/stepmania_complete_cdrom_questions_crosspost_from/)). Not much luck in all three places; hope someone here who sees this knows something.
